.circleci/config.yml

```diff
@@ -18,8 +18,21 @@ test-defaults: &test-defaults
 
     - run: |
         apt-get update
+        echo "keyboard-configuration keyboard-configuration/layoutcode string us" | debconf-set-selections
```

Comment:
Please add a comment for this line - I'm not sure how it is related to GL rendering